updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc

```diff
@@ -114,24 +114,27 @@ and must be no more than 63 characters`, try creating the Update Service with a
 ====
 
 // Configuring the Cluster Version Operator (CVO)
-include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
+include::modules/update-service-configure-cvo.adoc[leveloffset=+1]
 
 [NOTE]
 ====
 See xref:../../../networking/enable-cluster-wide-proxy.adoc#enable-cluster-wide-proxy[Configuring the cluster-wide proxy] to configure the CA to trust the update server.
 ====
 
+// Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+include::modules/configure-cvo-retrieve-image-signatures.adoc[leveloffset=+2]
+
 [id="next-steps_updating-restricted-network-cluster-osus"]
 == Next steps
 
 Before updating your cluster, confirm that the following conditions are met:
 
 * The Cluster Version Operator (CVO) is configured to use your locally-installed OpenShift Update Service application.
-* The release image signature config map for the new release is applied to your cluster.
+* Your cluster is able to retrieve release image signatures for the update target release images.
 +
 [NOTE]
 ====
-The release image signature config map allows the Cluster Version Operator (CVO) to ensure the integrity of release images by verifying that the actual image signatures match the expected signatures.
+Release image signatures allow the Cluster Version Operator (CVO) to ensure the integrity of release images by verifying that the actual image signatures match the expected signatures.
```

Comment:
It's hard for me to understand this text :( Any one have any idea to make it easy to understand? cc @jiajliu @shellyyang1989

Reply:
I can rephrase this sentence if it's hard to understand. I think it's explaining how the CVO uses release image signatures to verify that the release images you are trying to apply during an update match the images that Red Hat serves (and thus haven't been modified by someone else).

Maybe the note could be rephrased to this:
```suggestion
The Cluster Version Operator (CVO) uses release image signatures to ensure that release images have not been modified, by verifying that the release image signatures match the expected result.
```

Let me know if that rephrasing would make more sense to you.

Reply:
Cool! Right now it is very clear for me. Thank you @skopacz1